packages/hydrogen-react/src/customer-account-api-types.d.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * THIS FILE IS AUTO-GENERATED, DO NOT EDIT
- * Based on Customer Account API 2024-01
+ * Based on Customer Account API unstable
```

Comment:
Do NOT merge this in. The types are added here to see what type could still be failing